cjs/config.d.ts

```diff
@@ -0,0 +1,26 @@
+export interface ConfigInterface {
```

Comment:
Is there a reason why we added cjs and esm folders (they are build output so they should be in .gitignore, right)?

Reply:
It can be both ways depending on how we publish to npm. Perhaps we should stick to publishing as we customarily did.

Removed